File: python/muse/algorithms/Sorting.py
from muse.algorithms.sorting import (
    BubbleSort,
    InsertionSort,
    MergeSort,
    QuickSort,
    SelectionSort,
)


def bubbleSort(arr: list) -> None:
    BubbleSort.sort(arr)


def insertionSort(arr: list) -> None:
    InsertionSort.sort(arr)


def mergeSort(arr: list) -> None:
    MergeSort.sort(arr)


def quickSort(arr: list) -> None:
    QuickSort.sort(arr)


def selectionSort(arr: list) -> None:
    SelectionSort.sort(arr)

File: rust/muse/src/algorithms/sorting.rs
pub mod bubble_sort;
pub mod insertion_sort;
pub mod merge_sort;
pub mod quick_sort;
pub mod selection_sort;

pub fn bubble_sort(arr: &mut [i32]) {
    bubble_sort::sort(arr);
}

pub fn insertion_sort(arr: &mut [i32]) {
    insertion_sort::sort(arr);
}

pub fn merge_sort(arr: &mut [i32]) {
    merge_sort::sort(arr);
}

pub fn quick_sort(arr: &mut [i32]) {
    quick_sort::sort(arr);
}

pub fn selection_sort(arr: &mut [i32]) {
    selection_sort::sort(arr);
}

#[cfg(test)]
pub(crate) mod tests;
